addons/project/static/src/components/project_right_side_panel/project_right_side_panel.js

```diff
@@ -60,8 +60,8 @@ export class ProjectRightSidePanel extends Component {
     get showProjectProfitability() {
         return !!this.state.data.profitability_items
             && (
-                this.state.data.profitability_items.revenues.data.length > 0
-                || this.state.data.profitability_items.costs.data.length > 0
+                this.state.data.profitability_items.revenues.data.length === 0
+                || this.state.data.profitability_items.costs.data.length === 0
             );
     }
```

Comment:
```suggestion
    get showProjectProfitabilityHelper() {
        const { costs, revenues } = this.state.data.profitability_items;
        return !costs.data.length || !revenues.data.length;
    }
```
I think it is a good opportunity to rework that part.
To check, but, as far as I understand your code, `this.state.data.profitability_items` is checked before calling the function, so no need to check again.
Also, renamed the method